cloud/scope/powervs_machine.go

```diff
@@ -354,11 +354,11 @@ func (m *PowerVSMachineScope) CreateMachine() (*models.PVMInstanceReference, err
 }
 
 func (m *PowerVSMachineScope) resolveUserData() (string, error) {
-	userData, userDataFormat, err := m.GetRawBootstrapDataWithFormat()
+	userData, err := m.GetRawBootstrapDataWithFormat()
 	if err != nil {
 		return "", err
 	}
-	if m.UseIgnition(userDataFormat) {
+	if m.UseIgnition() {
```

Comment:
can we check the condition `m.IBMPowerVSCluster.Spec.Ignition != nil` here directly?

Reply:
Used at couple of places so kept as it is.